patches/chrome-browser-resources-pdf-manifest.json.patch

```diff
@@ -14,7 +14,7 @@ index 6c3b296b945034786037f26342fd92b0826ca400..09d2424c4d9be6fe1efd1f08090d0875
    "mime_types": [
      "application/pdf"
    ],
--  "content_security_policy": "script-src 'self' 'wasm-eval' blob: filesystem: chrome://resources; object-src * blob: externalfile: file: filesystem: data:; plugin-types application/x-google-chrome-pdf",
-+  "content_security_policy": "script-src 'self' 'wasm-eval' blob: filesystem: chrome://brave-resources chrome://resources; object-src * blob: externalfile: file: filesystem: data:; plugin-types application/x-google-chrome-pdf",
+-  "content_security_policy": "script-src 'self' 'wasm-eval' blob: filesystem: chrome://resources; object-src * blob: externalfile: file: filesystem: data:",
++  "content_security_policy": "script-src 'self' 'wasm-eval' blob: filesystem: chrome://brave-resources chrome://resources; object-src * blob: externalfile: file: filesystem: data:",
```

Comment:
we have brave-resources in the pdf plugin? Are we sure this is needed?

Reply:
I'm not familiar enough with this to be able to answer that question, but I see @yrliou was the person [added this in the first place](https://github.com/brave/brave-core/commit/67431034b4cdd56804d2fb0e69fd41a07c99da69) so maybe it's a better question for her?